app/src/main/res/drawable/searchview_rounded_bg.xml

```diff
@@ -0,0 +1,4 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="#ffffff" /> <!-- Cambia el color de fondo si lo deseas -->
```

Comment:
delete these comments.

Reply:
That is not my correspondence.

Reply:
but it should not have a comment in a language other than Spanish.